handbook/rbac/rbac.md

```diff
@@ -0,0 +1,128 @@
+
+!!! note
+    If a user has a role that grants access to an application in any workspace that application will be available to the user at all times. If a user has not been granted access to an application in any workspace that application will be hidden from the navigation menu.
+
+Custom roles may be created, duplicated, modified, and deleted as needed. If a role is updated all users mapped to that role are affected.
+
+Due to the large number of privileges available SystemLink includes several *built-in* roles to facilitate getting started. Go to the SystemLink **Security** application to review the details on privileges granted for each role.
+
+- **Systems Maintainer**: This role has full access to systems, packages, assets, alarms, dashboards, and WebVIs. This role had read-only access to data and analysis procedures.
+
+- **Data Maintainer**: This role has full access to data (files, tag, and test results), as well as DataFinder, Data Preparation, Analysis Automation, dashboards, and WebVIs. This role has read only access to systems, packages, and feeds.
+
+- **Collaborator**: This role is read-only for all resources within SystemLink.
+
+- **Automated Agent**: This role does not have access to any SystemLink applications. While it has privileges to create data, and some cases modify data, it cannot delete data. This role has no access to systems, package repository, states, or analysis routines.
+
+!!! note
+    Built in roles cannot be modified. They can be duplicated and the duplicated roles may be renamed and the privileges modified. This constraint exists because NI may add or change privileges for built-in roles between releases. Users assigned these built-in roles will have their privileges automatically change upon upgrade. To opt out of this behavior assign your users custom roles.
+
+#### Server Administrator Role
+
+SystemLink includes a special **Server Administrator Role**. This role has exclusive access to the **Security** application and can modify workspaces and roles. This role has full access to all of SystemLink, its applications, data, systems, etc. **for every workspace**. This is the most permissive role in SystemLink and should be used by users who administer access control for SystemLink.
+
+The user *admin* created during the guided setup of **NI Web Server Configuration** is automatically assigned the **Server Administrator** role. This is done such that administrators can access the **Security** application during initial setup and configure access control via an identity provider. When this has been completed, NI encourages disabling the user created during guided setup and instead assigning the **Server Administrator** role to users backed by your identity provider.
```

Comment:
Guided setup doesn't need to be run again after configuring the server admins, and I initially thought this meant disabling the admin user as part of the initial setup - meaning no user would be able to map server admins. 

```suggestion
The user *admin* created during the guided setup of **NI Web Server Configuration** is automatically assigned the **Server Administrator** role. This is done such that administrators can access the **Security** application during initial setup and configure access control via an identity provider. NI encourages disabling the *admin* user in the **NI Web Server Configuration** after you have assigned the **Server Administrator** role to users backed by your identity provider.
```